<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_miniFilm</class>
 <widget class="QWidget" name="C_miniFilm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : rgb(255, 255, 255);
border-color: rgb(0, 85, 255);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>231</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0" columnstretch="0,0,0">
    <property name="verticalSpacing">
     <number>2</number>
    </property>
    <item row="5" column="1">
     <widget class="QLabel" name="lbl_genre">
      <property name="text">
       <string>inconu</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="lbl_T_annee">
      <property name="text">
       <string>Année:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="lbl_ico">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>75</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="2" rowspan="2">
     <widget class="QPushButton" name="btn_details">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.482412, y1:0.983, x2:0.493025, y2:0.239, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(0, 85, 255);
border-radius: 10px;
border-width: 1px;
border-style: solid;</string>
      </property>
      <property name="text">
       <string>Détails</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" rowspan="4" colspan="2">
     <widget class="QLabel" name="lbl_affiche">
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="lbl_T_note">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Note:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="lbl_titre">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
      </property>
      <property name="text">
       <string>Inconnu</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="lbl_note">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="lbl_annee">
      <property name="text">
       <string>inconnue</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="lbl_T_genre">
      <property name="text">
       <string>Genre:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>clik()</slot>
 </slots>
</ui>
